addons/survey/static/src/js/survey_choice_image_zoomer.js

```diff
@@ -0,0 +1,104 @@
+odoo.define('survey.choice_image_zoomer', function (require) {
+    'use strict';
+
+var publicWidget = require('web.public.widget');
+
+publicWidget.Widget.surveyChoiceImageZoomer = publicWidget.Widget.extend({
```

Comment:
"SurveyImageZoom" ?

"choice" does not really add much.
(Could rename everything associated too-.